lektor_s3.py

```diff
@@ -189,4 +192,23 @@ def publish(self, target_url, credentials=None, **extra):
             yield 'deleting %s' % f
         self.delete_batch(diff['delete'])
 
-
+        # should we invalidate cloudfront?
```

Comment:
Would be slightly nicer to stick this in a `def invalidate_cloudfront(self, distribution_id, changed)`, just to match the conventions of this file.